src/main/java/Model/City.java

```diff
@@ -0,0 +1,56 @@
+package Model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Data
+@Table(name = "city")
+@NoArgsConstructor
+public class City {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "city_id")
+    private int id;
+    @Column(name = "city_name")
+    private String cityName;
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "City")
```

Comment:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "org.hibernate.SessionFactory.openSession()" because the return value of "Configuration.HibernateSessionFactoryUtil.getSessionFactory()" is null
	at Service.impl.EmployeeDAOImpl.addEmployee(EmployeeDAOImpl.java:16)
	at Application.main(Application.java:14)

Reply:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "org.hibernate.SessionFactory.openSession()" because the return value of "Configuration.HibernateSessionFactoryUtil.getSessionFactory()" is null
	at Service.impl.EmployeeDAOImpl.getEmployeeById(EmployeeDAOImpl.java:35)

Reply:
то что вы написали не связано ведь с openSession

Reply:
тут трабл какой то в HibernateSessionfactoryUtil